js/hotgraphicView.js

```diff
@@ -282,6 +285,8 @@ define([
         },
 
         preRemove: function() {
+            if (!this.completionEvent) return;
```

Comment:
When a Hotgraphic is re-rendered as a Narrative, `this.completionEvent` is `undefined`. This removes all events bound to `this.model`.